src/locales/fr.json

```diff
@@ -210,9 +210,9 @@
     },
     "settings": {
       "participation": "Participer à l'objectif “Aller au travail à vélo”",
-      "showAlerter": "Afficher l'alerter de l'objectif vélo",
-      "hideOnboarding": "Masquer l'embarquement d'objectif vélo",
-      "showAlerterSuccess": "Afficher l'alerte de succès de l'objectif vélo"
+      "showAlerter": "Objectif vélo : afficher l'alerter",
+      "hideOnboarding": "Objectif vélo : Masquer l'onboarding",
+      "showAlerterSuccess": "Objectif vélo : Afficher l'alerte de succès"
```

Comment:
nit: this should also be changed in the English file

Reply:
A l'occasion :p je doute que les "admins" utilisent la version US là c'est juste du rapide changement pour y voir plus clair en FR